server/jamovi/server/formatio/csv.py

```diff
@@ -251,7 +251,7 @@ def ruminate(self):
             self._unique_values = list(map(lambda x: int(x), self._unique_values))
             self._unique_values.sort()
             for level in self._unique_values:
-                self._column.append_level(level, str(level))
+                self._column.append_level(level, str(level), str(level))
```

Comment:
these guys can be simplified once `append_level()` has a default value for the third argument.